pwndbg/next.py

```diff
@@ -88,6 +89,16 @@ def break_next_call(address=None):
         if capstone.CS_GRP_CALL in ins.groups:
             return ins
 
+def break_call_regex(regex=''):
+    while pwndbg.proc.alive:
+        ins = break_next_call()
+
+        if not ins:
+            break
+
+        if re.search(regex, ins.symbol):
```

Comment:
In either case, it seems like this case can be easily handled at runtime.  Assuming that the user tells us that they want to stop at the next call to `printf`, ideally it'd work like:

```
pwndbg> nextcall printf
```

Note the lack of regex.  We can use regex internally to match e.g. `(_*)(printf)(_*)(@plt)`.